proto/v1/schema_design_service.proto

```diff
@@ -63,6 +63,13 @@ service SchemaDesignService {
       body: "*"
     };
   }
+
+  rpc StringifyMetadata(StringifyMetadataRequest) returns (StringifyMetadataResponse) {
```

Comment:
How about moving it to sql_service.proto, staying close to its friends Pretty()?

Reply:
Done